index.md

```diff
@@ -1,34 +1,25 @@
+Anbox Cloud supports x86 and Arm64 hardware, providing the same set of features for both architectures. 
 
-Anbox Cloud uses system containers to emulate Android systems. This method results in an isolation and security level similar to virtual machines with an overhead as low as process containers. Therefore, Anbox Cloud can provide at least twice the container density compared to other Android emulation solutions, and can serve up to 100 Android instances per server.
+Since Anbox Cloud uses containerisation to emulate Android systems, you can achieve the isolation and security level of a virtual machine without the associated overhead. Therefore, compared to other Android emulation solutions, Anbox Cloud can provide at least twice the container density and can serve up to 100 Android instances per server.
 
-You should consider using Anbox Cloud if you want to deliver mobile applications that don't depend on the capabilities of your user's devices and that offload compute, storage and energy-intensive applications from the end device to the cloud. Typical use cases include mobile game streaming services, corporate application streaming, application automation and Android application testing.
+Due to its highly scalable nature and performance optimisation, delivering device-agnostic mobile applications becomes very easy. Popular use cases of Anbox Cloud include mobile game streaming services, corporate application streaming, application automation and testing.
 
+# In this documentaion
 | | |
 |--|--|
 |  [Tutorials](https://discourse.ubuntu.com/t/tutorials/28826)</br>  Get started - a hands-on introduction to Anbox Cloud for new users </br> |  [How-to guides](https://discourse.ubuntu.com/t/how-to-guides/28827) </br> Step-by-step guides covering key operations and common tasks |
 |  [Explanation](https://discourse.ubuntu.com/t/explanation/28829) </br> Concepts - discussion and clarification of key topics, architecture  | [Reference](https://discourse.ubuntu.com/t/reference/28828) </br> Technical information - specifications, APIs |
 
-## What's new
-
-Along with bug fixes and general improvements, Anbox Cloud 1.18 includes the following changes:
-
-* Streaming improvements
-* Support for the Android container to use a custom Android ID
-* Updated Mesa driver 
-* Various dashboard improvements
-
-For detailed information, see the [Release notes](https://discourse.ubuntu.com/t/release-notes/17842).
-
 ## Project and community
 
 Anbox Cloud is a Canonical product. It originally grew out of the Anbox open-source project, but its code base is now completely independent.
 
 - [Get support through Ubuntu Pro](https://ubuntu.com/support)
 - [Join the Discourse forum to ask questions](https://discourse.ubuntu.com/c/anbox-cloud/49)
 - [Release roadmap](https://discourse.ubuntu.com/t/release-roadmap/19359)
+- [Report issues with Anbox Cloud](https://discourse.ubuntu.com/t/how-to-troubleshoot-anbox-cloud/17837)
```

Comment:
This sounds more like a troubleshooting rather than an issue report.
[Report issues with Anbox Cloud](https://bugs.launchpad.net/anbox-cloud/+bugs) looks reasonable to me

Reply:
I have included both links now, the idea was to increase the prominence of the troubleshooting section as well.